components/tidb_query/src/expr/builtin_string.rs

```diff
@@ -1882,34 +1895,30 @@ mod tests {
                 ],
                 Datum::Bytes("abc,CAFÉ,数据库".as_bytes().to_vec()),
             ),
-            (
-                vec![
-                    Datum::Bytes(b",".to_vec()),
-                    Datum::Bytes(b"abc".to_vec()),
-                    Datum::Null,
-                    Datum::Bytes(b"defg".to_vec()),
-                ],
-                Datum::Bytes(b"abc,defg".to_vec()),
-            ),
```

Comment:
Well, I think you don't need to delete them. Your new tests covers different cases from this.

Reply:
I reverted that change